File: Praktikum Pemrograman I Modul 4/Praktikum I Modul4  07-11-2023/Python/PRAK405-2310817110015-AKHMAD_CHAIDAR_ANANDA.py
# input
baris, kelipatan = map(int, input().split())

# deklarasi variabel hasil bernilai nol
hasil = 0

# perulangan
for i in range(1, baris + 1):
    
    # variabel temp akan tetap bernilai nol
    # variabel sementara yang menyimpan hasil penjumlahan dari setiap kelipatan
    temp = 0
    
    # perulangan kedua
    while (i > 0):

        temp += i * kelipatan
        hasil += i * kelipatan

        # kondisi
        if (i == 1):
            print(f"({i} * {kelipatan})", end=' ')
        else:
            print(f"({i} * {kelipatan}) +", end=' ')

        i -= 1
    # output hasil sementara dari setiap penjumlahan
    print(f"= {temp}")

# output hasil dari setiap penjumlahan
print(f"{hasil}")

File: Praktikum Pemrograman I Modul 4/Praktikum I Modul4  07-11-2023/C/PRAK405-2310817110015-AKHMAD_CHAIDAR_ANANDA.c
#include<stdio.h>

int main(){

    // deklarasi variabel
    int baris, kelipatan, hasil, temp;

    // input
    scanf("%d %d", &baris, &kelipatan);

    // deklarasi variabel perulangan
    int i, j;
    // nested loops
    for(i = 1; i <= baris; i++){
        
        // variabel temp akan tetap bernilai nol
        // variabel sementara yang menyimpan hasil penjumlahan dari setiap kelipatan
        temp = 0;
        // perulangan
        for(j = i; j > 0; j--){
            
            temp += j * kelipatan;
            hasil += j * kelipatan;
            
            // kondisi
            if (j == 1){
                printf("(%d * %d) ", j, kelipatan);
            } else {
                printf("(%d * %d) + ", j, kelipatan);
            }
        
        }
        // output hasil sementara dari setiap penjumlahan
        printf("= %d\n", temp);

    }
    
    // output hasil dari setiap penjumlahan
    printf("%d", hasil);

    return 0;
}
/*
for(i = 1; i <= a; i++){
        printf("%di ", i);
        //for(j = 1; j <= i; j++){
            
            for(j = 1; j < i; j++){
                printf("%dj ", j);
                for(k = j; k < j; ++k);
                k += 1;;
                printf("%djj ", j);
            }
            //j += 1;
        //}
        printf("\n");
    }
*/


/*
for(i = 1; i <= a; i++){
        printf("%d. ", i);
        for(j = 1; j < i; j++){
            printf("%d,", j);
        }
        printf("\n");
    }
*/

/*
for(i = 1; i <= a; i++){
        //printf("%d", i);
        for(j = 1; j <= i; j++){
            printf("(%d * %d)", i, b);
        }
        printf(" = %d\n", hasil);
    }
*/